test/test_routes.py

```diff
@@ -298,27 +298,27 @@ def test_get_blocked_paths(self):
         io.close()
         os.remove(tmp_file.name)
 
-    def test_get_file_created_by_app(self):
+    def test_get_file_created_by_app(self, test_client):
         app, _, _ = gr.Interface(lambda s: s.name, gr.File(), gr.File()).launch(
             prevent_thread_lock=True
         )
         client = TestClient(app)
+        with open("test/test_files/alphabet.txt") as f:
+            file_response = test_client.post("/upload", files={"files": f})
         response = client.post(
             "/api/predict/",
             json={
                 "data": [
                     {
-                        "data": media_data.BASE64_IMAGE,
-                        "name": "bus.png",
-                        "is_file": False,
-                        "size": len(media_data.BASE64_IMAGE),
+                        "path": file_response.json()[0],
+                        "size": os.path.getsize("test/test_files/alphabet.txt"),
```

Comment:
Changes this to upload a file instead, since the base64 thing gives validation errors now.